index.css

```diff
@@ -697,21 +688,17 @@ form {
   padding-bottom: 50px;
 }
 
-/* form validation */
-
-.form-submission-error {
-  background: azure;
-  color: red;
-  margin: 10px 0;
-  width: 105%;
+.img_mobi_modal {
```

Comment:
- [ ] [Optional] You can give the popup more size, so it can be as close as possible to the Figma design

|Your design|Figma design|
|---|---|
|![Screenshot_20220520_070259](https://user-images.githubusercontent.com/25227022/169463124-286e0386-120a-4e80-8374-4cfa52bfe84f.png)|![Screenshot_20220520_070158](https://user-images.githubusercontent.com/25227022/169462969-e179c6f0-69d8-447d-b1d5-365b26043039.png)|